src/World/locations.js

```diff
@@ -27,9 +30,10 @@ setup.initMiscLocations = () => {
         return `a rocky cave. ${readout} <blockquote>The cave is home to ${contents}.</blockquote>`
       }
     },
-    'a hole under a large tree': {
+    {
+      summary: 'a hole under a large tree',
       available: ['forest'],
-      function: (town, biome) => {
+      function: (_, biome) => {
```

Comment:
What's the thinking behind this?

Reply:
It's a pretty common convention to use underscores for unused leading parameters, as you can't remove them entirely.

It prevents linter/TS complaining about it as well.